src/features/campaigns/l10n/messageIds.ts

```diff
@@ -96,7 +96,6 @@ export default makeMessages('feat.campaigns', {
     },
     manager: {
       label: m('Project manager'),
-      selectSelf: m('Set yourself as manager'),
```

Comment:
Thank you! I think it should be possible to delete the entire `manager` section, right? It doesn't seem like it's used anywhere.